pages/settings/index.vue

```diff
@@ -29,12 +29,13 @@
   </div>
 
   <div v-else class="lhs-bg w-100 h-100">
-    <overlay-scrollbars class="narrow-scrollbar h-100"
-                        :options="{scrollbars: {autoHide: 'leave', autoHideDelay: 0}}">
+    <simplebar class="simplebar narrow-scrollbar h-100"
+               data-simplebar-auto-hide="false" data-simplebar-force-visible="true"
+               :options="{scrollbars: {autoHide: 'leave', autoHideDelay: 0}}">
```

Comment:
The `options` attribute does nothing on `<simplebar/>` components
```suggestion
               >
```